Source/Fuse.Nodes/Node.uno

```diff
@@ -156,5 +156,22 @@ namespace Fuse
 			}
 			return null;
 		}
+		
+		public T FindByType<T>() where T : class
```

Comment:
I loosened it because I need to look for things that aren't nodes. In particular I'm looking for an interface. Even though I know what I find will be a node, the type I'm looking for isn't.